src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -17,12 +17,10 @@
     public User() {
     }
 
-    public User(Long id, String username, String status, Long userKey, List<Order> orders) {
-        this.id = id;
+    public User(String username, String status, Long userKey) {
```

Comment:
dlaczego wyrzuciliśmy pozostałe parametry z konstruktora? pewnie coś za tym stało ale chciałbym zrozumiec :)

Reply:
osoba (nie pamiętam kto i nie chce mi się sprawdzać) która robiła testy operacji CRUD na encji user twierdziła że są z tym jakieś problemy więc też to usunąłem :) w gałęzi master w tym momencie też tak jest.